conf/pacific/rados/6-node-cluster-6-clients.yaml

```diff
@@ -0,0 +1,94 @@
+# Rados tier-2 test configuration, small test config for basic tests
+# Deployment for all the ceph daemons , with 3 mon's, 3 mgr's, and 15 OSD daemons
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        networks:
+          - provider_net_cci_12
+        role:
+          - _admin
+          - mon
+          - mgr
+          - installer
+          - node-exporter
+          - alertmanager
+          - grafana
+          - prometheus
+          - crash
+      node2:
+        networks:
+          - provider_net_cci_12
+        role:
+          - mon
+          - mgr
+          - mds
+          - rgw
+          - node-exporter
+          - alertmanager
+          - crash
+      node3:
+        networks:
+          - provider_net_cci_12
+        role:
+          - osd
+          - node-exporter
+          - crash
+        no-of-volumes: 5
+        disk-size: 25
+      node4:
+        networks:
+          - provider_net_cci_12
+        role:
+          - osd
+          - node-exporter
+          - crash
+        no-of-volumes: 5
+        disk-size: 25
+      node5:
+        role:
+          - osd
+          - node-exporter
+          - crash
+        no-of-volumes: 5
+        disk-size: 25
+      node6:
+        networks:
+          - provider_net_cci_12
+        role:
+          - mon
+          - mgr
+          - mds
+          - rgw
+          - node-exporter
+          - crash
+      node7:
+        networks:
+          - provider_net_cci_8
+        role:
+          - client
+      node8:
+        networks:
+          - provider_net_cci_8
+        role:
+          - client
+      node9:
+        networks:
+          - provider_net_cci_9
+        role:
+          - client
+      node10:
+        networks:
+          - provider_net_cci_9
+        role:
+          - client
+      node11:
+        networks:
+          - provider_net_cci_7
+        role:
+          - client
+      node12:
```

Comment:
There are total 12 nodes exist in the file but the file name is "6-node-cluster-6-clients.yaml"

Reply:
This would be a special suite, were we have multiple clients, spread across subnets, wich would be used only for CIDR testing.

Naming it 12 node suite would give ppl assumption that we have that many nodes as part of cluster, whereas, here, only 6 are part of cluster, and 6 are clients.

Hence the 6-6 naming convention.  Let me know if you think we can have a better convention here.